src/components/Form/DynamicInput.tsx

```diff
@@ -87,6 +88,14 @@ const DynamicInput = (props: DynamicInputProps) => {
                     </Suspense>
                 );
             }
+        case "sourceEditor":
```

Comment:
thought: eventually we want to have a single option for "visualEditor" or something like that and pass a schema and config to render the editor properly. Please make sure you take that into consideration for the technical design you made for a future change.